website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -20,13 +20,25 @@ To add an application to authentik and have it display on users' **My applicatio
 
 2. Click **Create with Wizard**. (Alternatively, use our legacy process and click **Create**. The legacy process requires that the application and its authentication provider be configured separately.)
 
-3. In the **New application** wizard, define the application details, the provider type and configuration, and then click **Submit**.
+3. In the **New application** wizard, define the application details, the provider type, bindings for the application.
 
-4. To manage the display of the new application on the **My applications** page, you can optionally define the bindings for a specific policy, group, or user. Note that if you do not define bindings, then all users have access to the application, For more information, refer to [authorization](#authorization).
+- **Application**: provide a name, an optional group for the type of application, the policy engine mode, and optional UI settings.
+
+- **Choose a Provider**: Select the provider types for this application.
+
+- **Configure a Provider**: Provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and any additional required configurations.
+
+- **Configure Bindings**: To manage the display of the new application on the **My applications** page, you can optionally define [bindings](../flows-stages/bindings/index.md) for a specific policy, group, or user. To do so in the Wizard, click **Bind existing policy/group/user** to add a binding. You can select an existing policy binding, or create a new binding specifically for a group or user. For example, if you select **User** and then choose an existing user from the drop-down menu, you create a new binding between the user and this specific application. Note that if you do not define any bindings, then all users have access to the application. For more information, refer to [authorization](#authorization).
```

Comment:
I worry that this confuses the issue for experienced users.  I don't know that there's a need to explain the difference between Users, Groups, and other Policies here.  That's the problem with having "Policy" Policy bindings.  Maybe we should call them something else, I dunno, "Custom" bindings.

Reply:
It is a bit confusing, precisely because there is not an exact parallel between the three tab options of Policy/Group/User. They behave differently. With Policy, you have to select an existing policy bindings. With Group or User, you are choosing to create a new binding.

We have to document how the product works, but tell me where the wording could change to make it more clear @kensternberg-authentik ?

Reply:
I am interested in the idea of calling them something else... maybe Application bindings. And then once you say you want an application binding, you choose via radio buttons if you want to bind a Group, a User, or an existing Policy binding. So it ends up with same result, but the beginning part, with the three tabs, might not be so confusing.